pkg/jobs/adopt.go

```diff
@@ -248,8 +248,23 @@ func (r *Registry) runJob(
 	defer cleanup()
 	spanName := fmt.Sprintf(`%s-%d`, typ, job.ID())
 	var span *tracing.Span
-	ctx, span = r.ac.AnnotateCtxWithSpan(ctx, spanName)
+
+	// Wrap the context with a root span that is tied to the execution of the
```

Comment:
I'd drop the "Wrap the context" verbiage, lets just say "Create a new root span to trace the execution of the current instance of job...".

Reply:
done.